### DISCUSSION TITLE
Docker

### DISCUSSION THREAD

**Discussion by Dom1w** [ZUGFeRD/mustangproject#514] [Ideas]
Hi,
I wanted to ask, if it makes sense to add a `Dockerfile` to the repo, that would build a simple webservice with the basic functions from https://www.mustangproject.org/commandline

I right now made one myself, cause I needed the xml to pdf function, but I think it would be very beneficial to have one directly in the repo to just run `docker build .`

**Comment by jstaerk** [comment]
Hello,

There is a dockerfile available: the in house version of our commercial product https://www.mustangproject.org/server/ . 
We also thought it would be easy but it soon turned out that much more could and needed to be done :-)

kind regards
Jochen

**Reply by Dom1w** [reply]
So I understand you right, that you don't want me add the webservice + dockerfile here, cause it would invalidate your commercial product?

**Reply by jstaerk** [reply]
I just recommend PRing to the correct repo, feel free to be the first one actually contributing to https://github.com/ZUGFeRD/mustangserver
